src/Task/Controllers/Task_Controller.php

```diff
@@ -541,7 +541,7 @@ public static function delete_task( $data ) {
             ->first();
 
         $resource = new Item( $task, new Task_Transformer );
-        $resource = $this->get_response( $resource );
+        $resource =  self::get_response( $resource );
```

Comment:
You could write it this way, as `$self` is already declared in this method and also `get_response()` is not a static method.

```php
$self->get_response( $resource );
```
Please remove also extra space before starting `$self` according to `phpcs`